src/bun.js/webcore/blob.zig

```diff
@@ -3423,12 +3431,192 @@ pub const Blob = struct {
         return JSValue.jsBoolean(bun.isRegularFile(store.data.file.mode) or bun.C.S.ISFIFO(store.data.file.mode));
     }
 
+    fn isS3(this: *Blob) bool {
+        if (this.store) |store| {
+            if (store.data == .file) {
```

Comment:
Maybe we could add a `s3` or `remote` to the Store instead?

Reply:
`stora.data== .s3` is what I was planing for today, because we also need to store the options passed.